content/departments/sales/tools/referral_and_reseller_policy.md

```diff
@@ -0,0 +1,94 @@
+## Referral Program Summary
+
+### Program Purpose:
+
+The Sourcegraph Business Development team wants to enlist the partner community to accelerate Sourcegraph’s growth. The primary impetus of referrals should be that Sourcegraph provides meaningful business benefits to the end customers. Referral payments are important part of certain partners’ business models and Sourcegraph will enthusiastically pay them for accelerating our customer acquisition.
+
+### Partner Nomination:
+
+Potential referral partners should be referred to the Head of Business Development
+
+### Agreement:
+
+The partner referral agreement is available [here](https://docs.google.com/document/d/1UnEJ2Lzw0slRkquHWjjpsO0KN7SVfZgy/edit). Note that this will eventually be architected to stand as an addendum to a standard partnership agreement.
```

Comment:
The link needs to have a descriptive accessible name).

Please see this comment: https://github.com/sourcegraph/handbook/pull/5542#discussion_r1018559975

> Links need to have a descriptive name for accessibility. Instead of using the word "here", rephrase the sentence and link the part of the sentence that describes the document, e.g.
> 
> > **Agreement**: Use our [partner referral agreement template](https://docs.google.com/document/d/1UnEJ2Lzw0slRkquHWjjpsO0KN7SVfZgy/edit). Note that this will eventually be architected to stand as an addendum to a standard partnership agreement.